drivers/modem/quectel-bg9x.c

```diff
@@ -1024,13 +1065,11 @@ static int modem_setup(void)
 				       &mdata.rssi_query_work,
 				       K_SECONDS(RSSI_TIMEOUT_SECS));
 
-	/* Once the network is ready, activate PDP context. */
-	ret = modem_cmd_send(&mctx.iface, &mctx.cmd_handler,
-			     NULL, 0U, "AT+QIACT=1", &mdata.sem_response,
-			     MDM_CMD_TIMEOUT);
-
-	/* Retry or Possibly Exit. */
+	/* Once the network is ready, we try to open the PDP context. */
+	ret = modem_pdp_context_activate();
 	if (ret < 0 && init_retry_count++ < MDM_INIT_RETRY_COUNT) {
+		LOG_ERR("Error opening modem with pdp context n°1");
+		init_retry_count++;
```

Comment:
Why do you increment `init_retry_count` second time?

Also: I think we can remove ` n°1` from the error message for now.